movie_rec/recommender.py

```diff
@@ -1,39 +1,73 @@
 import json
 from optparse import OptionParser
+import psycopg2
 
 class MovieRecommender(object):
-    def __init__(self, hypothesis_path, positive, negative):
+    def __init__(self, hypothesis_path, target_user):
         self.predictions = self.load_hypothesis(hypothesis_path)
-        self.liked_movies = self.load_annotations(positive)
-        self.disliked_movies = self.load_annotations(negative)
+        self.liked_movies = self.load_movies('1', target_user)
+        self.disliked_movies = self.load_movies('-1', target_user)
         self.seen_movies = set(self.liked_movies + self.disliked_movies)
-        
+	self.movie_mapping = self.load_mapping()
+
+    def load_mapping(self):
+        conn = psycopg2.connect('dbname=movie_rec')
+        cur = conn.cursor()
+        query = "select movie_id, title from movies"
+        cur.execute(query)
+        results = cur.fetchall()
+        cur.close()
+        conn.close()
+        return dict([
+            (result[0], result[1])
+            for result in results
+        ])
+
+
+    def load_movies(self, classification, target_user):
```

Comment:
yeah. when I was working on this locally, wanted to keep the recommendation part separate from the training part. instantiating the movie_reviews object takes a decent amount of time since it also does all the reviews, but when we do this on the backend, I don't think these will be separate so can use the like/dislike movies already in the movie_reviews object instead of requerying.